src/test/scala/com/microsoft/ml/spark/explainers/ImageExplainersSuite.scala

```diff
@@ -0,0 +1,159 @@
+    .setSuperpixelCol("superpixels")
+    .setMetricsCol("r2")
+    .setInputCol("image")
+    .setCellSize(100)
+    .setModifier(20)
+    .setNumSamples(3)
+
+  lazy val greyhoundImageLocation: String = {
+    val loc = "/tmp/greyhound.jpg"
+    val f = new File(loc)
+    if (f.exists()) {
+      f.delete()
+    }
+    FileUtils.copyURLToFile(new URL("https://mmlspark.blob.core.windows.net/datasets/LIME/greyhound.jpg"), f)
+    loc
+  }
+
+  val imageDf: DataFrame = spark.read.image.load(greyhoundImageLocation)
+}
+
+class ImageSHAPExplainerSuite extends ImageExplainersSuite
+  with TransformerFuzzing[ImageSHAP] {
+
+  import spark.implicits._
+
+  test("ImageKernelSHAP can explain a model locally") {
+    val (image, superpixels, shapValues, r2) = shap
+      .transform(imageDf)
+      .select("image", "superpixels", "weights", "r2")
+      .as[(ImageFormat, SuperpixelData, Seq[SV], SV)]
+      .head
+
+    // println(shapValues)
+    // println(r2)
+
+    // R2 should be almost 1.
+
+    val spStates = shapValues.head.toBreeze(1 to -1).map(_ >= 0.05).toArray
+    // println(spStates.count(identity))
+
+    // Uncomment the following lines lines to view the censoredImage image.
+    // import com.microsoft.ml.spark.io.image.ImageUtils
+    // import com.microsoft.ml.spark.lime.Superpixel
+    // import java.awt.image.BufferedImage
+    // val originalImage = ImageUtils.toBufferedImage(image.data, image.width, image.height, image.nChannels)
+    // val censoredImage: BufferedImage = Superpixel.maskImage(originalImage, superpixels, spStates)
+    // Superpixel.displayImage(censoredImage)
+    // Thread.sleep(100000)
+  }
+
+  override def testObjects(): Seq[TestObject[ImageSHAP]] = Seq(new TestObject(shap, imageDf))
+
+  override def reader: MLReadable[_] = ImageSHAP
+}
+
+class ImageLIMEExplainerSuite extends ImageExplainersSuite
+  with TransformerFuzzing[ImageLIME] {
+
+  import spark.implicits._
+
+  test("ImageLIME can explain a model locally for image type observation") {
+    val (image, superpixels, weights, r2) = lime
+      .transform(imageDf)
+      .select("image", "superpixels", "weights", "r2")
+      .as[(ImageFormat, SuperpixelData, Seq[SV], SV)]
+      .head
+
+    // println(weights)
+    // println(r2)
+
+    val spStates = weights.head.toBreeze.map(_ >= 0.2).toArray
+    // println(spStates.count(identity))
+
+    // Uncomment the following lines lines to view the censoredImage image.
+    // import com.microsoft.ml.spark.io.image.ImageUtils
+    // import com.microsoft.ml.spark.lime.{Superpixel, SuperpixelData}
+    // import java.awt.image.BufferedImage
+    // val originalImage = ImageUtils.toBufferedImage(image.data, image.width, image.height, image.nChannels)
+    // val censoredImage: BufferedImage = Superpixel.maskImage(originalImage, superpixels, spStates)
+    // Superpixel.displayImage(censoredImage)
+    // Thread.sleep(100000)
+  }
+
+  // Do not run in CI pipeline due to high memory needs
```

Comment:
it looks like this is run anyway?  it's not disabled?

Reply:
Actually it shouldn't be disabled. I removed the comment.